lib/order.rb

```diff
@@ -0,0 +1,27 @@
+require_relative 'menu'
+require_relative 'total'
+
+
+class Order
+  attr_reader :menu, :total
+
+  def initialize(menu = Menu.new, total = Total.new)

```

Comment:
I'd say the `Checkout` should also be a direct dependency of `Order` rather than of `Total` — maybe the naming could be changed as well to make the intent clearer. I'd suggest renaming `Order` to `OrderBuilder` maybe, and `Total` to `Order` (since it represents the *actual* order selection)